### DISCUSSION TITLE
user-guide/why-use-zotero

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#85] [文档评论区]
# user-guide/why-use-zotero

本节是官方文档的翻译。 Zotero 是什么 Zotero 是一款开源的文献管理软件。它可以用于抓取、存储、管理、阅读、引用文献（包含书籍与论文）。使用 Zotero 将使你的学术生产效率大增。 你是否仍在花费大量时间整理阅读过和刚下载的文献？; 你是否仍在为找不到自己记忆中那篇文献而苦恼？; 你是否还在焦虑刚刚读完文献却脑袋空空？; 你是否还在手打插...

https://zotero-chinese.com/user-guide/why-use-zotero.html

**Comment by XuhengZhu** [comment]
怎么没与评论呢，加油！

**Reply by northword** [reply]
大概是因为评论区刚开没多久 🙌

**Comment by Jerryzhao83** [comment]
CNKI的E-Study，最近刚刚更新，发现它限制了只能管理1万条文献。超过1万条文献自动删除最早添加的。
非常不利于做持久、长期研究的学者使用。

**Reply by Jerryzhao83** [reply]
单机模式可以管理1万条。从来没有用过在线模式，不做评论。

**Reply by PangPangDog** [reply]
这个限制好坑爹啊

**Comment by jinger9** [comment]
献上我的膝盖！！

**Comment by TTGAcA** [comment]
你好，GB-T-7714-2005格式能否增加一个有DOI的样式，有些杂志要求使用GB-T-7714-2005格式但需要添加DOI

**Comment by rrr312** [comment]
hello

**Comment by mermerha** [comment]
加油加油

**Comment by xinfa1** [comment]
华为的硅基流动邀请码：qDLMR0ps  邀请链接：https://cloud.siliconflow.cn/i/qDLMR0ps
注：通过分享邀请链接或者在注册时候输入邀请码，每邀请一人，双方均可以获取2000w字符量，永久有效。欢迎互帮互助呀

**Comment by poterking** [comment]
功在当代,利在千秋!!!!
